src/config/file_info.rs

```diff
@@ -53,6 +54,14 @@ impl FileInfo<'_, '_, '_, '_> {
                 None
             };
             let mode = Self::get_mode(table, source, idx)?;
+            let caps = if let Some(caps) = table.get("caps") {
+                Some(
+                    caps.as_str()
+                        .ok_or(ConfigError::AssetFileWrongType(idx, "caps", "string"))?,
+                )
+            } else {
+                None
+            };
```

Comment:
Since you add `caps` at the end of the `FileInfo` struct, I would like this process to match that order as well.